bulk-user-export-tool/README.md

```diff
@@ -0,0 +1,49 @@
+# Bulk User Export Tool
+
+Java client that can export a bulk of users as a comma-separated values (.csv) using SCIM 2.0 APIs.
+
+## Table of contents
+
+- [Download and build](#download-and-build)
+- [Exporting user data from WSO2 IS](#exporting-user-data-from-wso2-is)
```

Comment:
Let's reorder the content.
Bring "Exporting user data from WSO2 IS" section to the top. A user should be able to use the binary to consume the tool, without needing to build it. If someone needs to build the tool(optionally with modification) then can give the steps how to do that.